salt/grains/core.py

```diff
@@ -1482,7 +1482,7 @@ def os_data():
     grains.update(_ps(grains))
 
     # Load additional OS family grains
-    if grains['os_family'] == "RedHat":
+    if grains['os_family'] in ['RedHat', 'SUSE']:
         grains['osmajorrelease'] = grains['osrelease'].split('.', 1)[0]
```

Comment:
No need to do the split? I can change that, but that's just how it was already done for redhat.